README.md

```diff
@@ -287,4 +287,6 @@ sh forward.sh
 # Troubleshooting
 ### RNS manager missmatch between networks
 Solution: switch back and forth a network on MetaMask/Nifty. If that does not work make sure you have setup correctly the network id in the RNS step.
-    
\ No newline at end of file
+    
+    
+    https://github.com/rsksmart/rif-notifier/tree/rgwy135
```

Comment:
lets remove this line and we can do the Trigger running in separate PR.

Reply:
done